src/components/Attachments/AttachmentCarousel/Pager/index.tsx

```diff
@@ -121,26 +147,38 @@ function AttachmentCarouselPager({items, renderItem, initialIndex, onPageSelecte
         [],
     );
 
+    const Content = useMemo(
```

Comment:
Rename this to something more descriptive (and lower case the variable name). Suggestion: `carouselItems`.

Reply:
Why is `useMemo()` being used here? Is there a problem with rendering the content inline down below like it was before?